sentry-kotlin-multiplatform/src/androidMain/kotlin/io/sentry/kotlin/multiplatform/extensions/SentryBreadcrumbExtensions.kt

```diff
@@ -0,0 +1,20 @@
+package io.sentry.kotlin.multiplatform.extensions
+
+import io.sentry.kotlin.multiplatform.AndroidSentryBreadcrumb
+import io.sentry.kotlin.multiplatform.SentryBreadcrumb
+import io.sentry.util.CollectionUtils
+
+fun SentryBreadcrumb.toAndroidBreadcrumb(): AndroidSentryBreadcrumb {
+    val androidBreadcrumb = AndroidSentryBreadcrumb()
+    androidBreadcrumb.message = this.getMessage()
+    androidBreadcrumb.type = this.getType()
+    androidBreadcrumb.category = this.getCategory()
+    val dataClone = CollectionUtils.newConcurrentHashMap(this.getData() as MutableMap<String?, Any>)
```

Comment:
Why do we use `newConcurrentHashMap` if `this.getData()` is never null? and if it is, your cast is going to fail.
The `key` can't be null either.